todo_app/e2e_test.py

```diff
@@ -0,0 +1,64 @@
+import os
+import pytest
+import requests
+from threading import Thread
+from selenium import webdriver
+#from todo_app.app import create_app
+#os.path.dirname(r'todo_app\app.py')
+from todo_app.app import create_app
+
+
+@pytest.fixture(scope='module')
+def create_trello_board():
+    board_id = create_trello_board()
+    os.environ['TOID'] = board_id
+    application = create_app()
+    return application
+    
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+# Create the new board & update the board id environment variable
+   # board_id = create_trello_board()
```

Comment:
We should make sure that these tests can be run against a fresh environment, rather than directly against your local environment that needs to be running at the same time, i.e. using this create_trello_board etc.